### PR TITLE
[Do not merge] Submit review request of Extension:ApprovedRevs

### PR DESCRIPTION
**Pull request by AmandaQuad**
The Phabricator login system is currently failing me, so I am making a PR for the ApprovedRevs extension.

@labster please review as soon as possible.

### CODE REVIEW

**Comment by NDKilla** [comment]
@1-2-1-2-3-Miraheze is there a reason not to use FlaggedRevs (which is already reviewed and installed?)

**Comment by 1-2-1-2-3-Miraheze** [comment]
@NDKilla FlaggedRevs (from the look of it on the MediaWiki page) is a lot more complicated, and neither I nor my sister want to deal with complicated right now. I didn't read through the entire extension entry, but that's my first impression of it.

File: LocalSettings.php [hunk 1]
```diff
@@ -703,6 +703,11 @@
 		'inazumaelevenwiki' => true,
 		'thelonsdalebattalionwiki' => true,
 	),
+	'wmgUseApprovedRevs' => array(
+		'default' => false
```
Comment (LegoFan4000) [inline LocalSettings.php]: there should be a comma after false

File: LocalSettings.php [hunk 2]
```diff
@@ -703,6 +703,11 @@
...
+		'extloadwiki' => true
+		'wikicanadawiki' => true
```
Comment (LegoFan4000) [inline LocalSettings.php]: there should be a comma after true

File: LocalSettings.php [hunk 3]
```diff
@@ -703,6 +703,11 @@
 		'inazumaelevenwiki' => true,
 		'thelonsdalebattalionwiki' => true,
 	),
+	'wmgUseApprovedRevs' => array(
+		'default' => false
+		'extloadwiki' => true
```
Comment (LegoFan4000) [inline LocalSettings.php]: there should be a comma after true

**Comment by LegoFan4000** [comment]
I reopened the associated task.

**Comment by NDKilla** [comment]
.-. this was supposed to be a review pr ;p
if we're talking about actual review of code, I think this is also missing a definition in LocalExtensions.php

**Comment by 1-2-1-2-3-Miraheze** [comment]
@NDKilla I did define the extension in LocalSettings.php as part of this pull request.

On a completely different note, I figured out why my Phabricator account is refusing to link. My IP address is hardblocked on Meta in addition to being anon-only globally blocked, and the hardblock is preventing my account from using OAuth applications.

**Comment by JohnFLewis** [comment]
The ticket as been re opened.

This is an incomplete change and the most effective repo to open it in would have been mediawiki.

**Pull request closed without merging**